### PR TITLE
feat: fingerprinting static asset files and updating internal referen…

### PR DESCRIPTION
**Pull request by wyseguyonline**
…ces to those files

### CODE REVIEW

**Comment by wyseguyonline** [comment]
@StommePoes @JasonLantz @umahaea

File: gulpfile.js [hunk 1]
```diff
@@ -41,19 +43,32 @@ gulp.task('lint', () => {
 });
 
 gulp.task('sass', () => {
-  gulp.src('./scss/elements.scss')
+  return gulp.src('./scss/elements.scss')
     .pipe(sass().on('error', sass.logError))
     .pipe(postcss([autoprefixer]))
-    .pipe(gulp.dest('./dist/css'))
+    .pipe(gulp.dest('./dist/temp'))
```
Comment (umahaea) [inline gulpfile.js]: @wyseguyonline - why do we change this path? This is failing all our tests, _ux-test-platform_ is reading the elements.css file from this location of elements /dist/css/elements.css....

**Comment by umahaea** [comment]
@wyseguyonline - In this branch, the __elements.css__ file is pushed to __css/dist/temp/elements.css__ 
The selenium tests are failing for this branch, because _ux-test-platform_ is reading the elements.css file from __/css/dist/elements.css__
https://travis-ci.org/Pearson-Higher-Ed/ux-test-platform/builds/151369348
https://travis-ci.org/Pearson-Higher-Ed/ux-test-platform/builds/151369984

File: scss/_variables.scss [hunk 2]
```diff
@@ -7,10 +7,11 @@
 // of bundled assets.
 
 $pe-icons-path: '/fonts/' !default;
-// Icon font filename without the extension.
+// Icon font filename need extensions in order to use find and replace.
```
Comment (StommePoes) [inline scss/_variables.scss]: ...yet you leave the next line without the extension.
```
$pe-icons-name: 'pearson-fa' !default;
``` 
Which is what I thought the comment was referring to.

Also, we're very likely going to dump most of the fonts later in preference to only woff and woff2.

**Comment by wyseguyonline** [comment]
@umahaea I have changed the directory the css files go to.  However, be advised that until the gulp build job (and its prerequisites) is complete, the CSS files will not be referencing the correct static assets (font and image files).

**Comment by JasonLantz** [comment]
my gulp is a little rusty, but this looks solid....

**Comment by umahaea** [comment]
I'm good with it, as it is not breaking the ci tests..

**Pull request merged**